packages/core/src/DreifussWysiwygEditor.tsx

```diff
@@ -137,20 +145,27 @@ export default function DreifussWysiwygEditor(props: EditorProps) {
     createHeadingPlugin({levels: 3}),
     createDeserializeMDPlugin(),
     createImagePlugin(),
+    searchHighlightPlugin,
     createNodeIdPlugin(),
-    createDndPlugin()
+    createDndPlugin(),
+    createExitBreakPlugin(optionsExitBreakPlugin),
+    createSoftBreakPlugin(optionsSoftBreakPlugin),
+    createResetNodePlugin(optionsResetBlockTypePlugin),
+    createSelectOnBackspacePlugin({allow: [ELEMENT_MEDIA_EMBED, ELEMENT_IMAGE]})
   ]
 
   return (
     <DndProvider backend={HTML5Backend}>
       <Plate
         id={props.id}
-        onChange={props.onChange}
+        onChange={val => props.onChange(val.map(({id, ...block}) => block))}
```

Comment:
What is this `id` ? And for what is it used?

Reply:
Drag and drop to work well, it adds  `id` key with unique value for every node in rich-text .. so it can use it while dragging them .. But I guess we won’t need the id prop to be saved in the db. So, on change, I am removing the id from the value.